cmd/kubeadm/app/cmd/options/constant.go

```diff
@@ -117,3 +117,9 @@ const FileDiscovery = "discovery-file"
 
 // ControlPlane flag instruct kubeadm to create a new control plane instance on this node
 const ControlPlane = "experimental-control-plane"
+
+// UploadCerts flag instruct kubeadm to upload certificates to kubeadm-certs
+const UploadCerts = "experimental-upload-certs"
+
+// SkipCertificateKeyPrint flag instruct kubeadm to skip printing of the default certificate key generated by 'kubeadm init'.
```

Comment:
Based on this comment I'm guessing kubeadm will never print the key if it's provided, but how would a user get the key if it's generated by kubeadm if they skip printing it?

Reply:
Yes, we should add this flag when we allow the user to provide the key. I'll update soon.